docs/tutorial/prismic-source-graphql-tutorial.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Using the Prismic with GraphQL Source Plugin
+---
+
+## How to create a blog capable of previews using data from Prismic
+
+### What’s contained in this tutorial?
+
+By the end of this tutorial, you’ll have done the following:
+
+- Learned how to configure the latest version of the Prismic source plugin
+- Fetch and render data from a Prismic repository
+- Create pages programmatically from a Prismic source
+- Configure previews for unpublished and edited documents
+
+### Prerequisites
+
+- Familiarity with Gatsby
+- A personal Prismic repository. You can follow this guide (simple user guide to come) if you want instructions in setting up a barebones repository for a minimalistic blog
```

Comment:
```suggestion
- A personal Prismic repository. You can follow this guide if you want instructions in setting up a barebones repository for a minimalistic blog
```
Do we need this?

Reply:
The tutorial is written around a specific data model that this guide walks you through on how to create. Link will be of course be provided